transformato/system.py

```diff
@@ -168,8 +168,9 @@ def _read_parameters(self, env: str) -> pm.charmm.CharmmParameterSet:
         parameter_files = tuple()
         l1 = f"{charmm_gui_env}/{tlc_lower}/{tlc_lower}.rtf"
         l2 = f"{charmm_gui_env}/{tlc_lower}/{tlc_lower}.prm"
+        l3 = f"{charmm_gui_env}/{tlc_lower}/{tlc_lower}.str"
 
-        for file_path in [l1, l2]:
+        for file_path in [l1, l2, l3]:
             if os.path.isfile(file_path):
                 parameter_files += (file_path,)
             else:
```

Comment:
it might be useful to have something like this:
```python
for check in checks:
  if check:
      break
  else:
      print('check not sucessful')
      continue
else:
  raise RuntimeError('Couldn't find ligand parameters. Aborting.')
```
which raises an error if no parameter file is found.